data/disney

```diff
@@ -103,7 +108,6 @@ disneytvajobs.com
 disneyworld-go.com
 dssott.com
 go-disneyworldgo.com
-go.com
```

Comment:
What happens to `cdn.registerdisney.go.com`?
And why is the whole `go.com` being deleted?

Reply:
> What happens to `cdn.registerdisney.go.com`?
> And why is the whole `go.com` being deleted?

I need to know the answer before I give this pr approval.